openqaoa/backends/qaoa_backend.py

```diff
@@ -43,7 +45,8 @@ def _backend_arg_mapper(backend_obj: QAOABaseBackend,
                         noise_model = None,
                         active_reset: Optional[bool] = None,
                         rewiring = None,
-                        qubit_layout = None):
+                        qubit_layout = None, 
+                        disable_qubit_rewiring: Optional[bool] = None):
```

Comment:
@shahidee44, I think the disable_qubit_rewiring argument is repeated. The argument `rewiring` used for the Pyquil backend does the same thing, no?